engine/src/flutter/lib/ui/painting.dart

```diff
@@ -52,45 +52,52 @@ Color _scaleAlpha(Color x, double factor) {
   return x.withValues(alpha: clampDouble(x.a * factor, 0, 1));
 }
 
-/// An immutable 32 bit color value in ARGB format.
+/// An immutable color value in ARGB format.
 ///
-/// Consider the light teal of the Flutter logo. It is fully opaque, with a red
-/// channel value of 0x42 (66), a green channel value of 0xA5 (165), and a blue
-/// channel value of 0xF5 (245). In the common "hash syntax" for color values,
-/// it would be described as `#42A5F5`.
+/// Consider the light teal of the [Flutter logo](https://flutter.dev/brand). It
+/// is fully opaque, with a red [r] channel value of `0.2588` (or `0x42` or `66`
+/// as an 8-bit value), a green [g] channel value of `0.6471` (or `0xA5` or
+/// `165` as an 8-bit value), and a blue [b] channel value of `0.9608` (or
+/// `0xF5` or `245` as an 8-bit value). In the common "hash syntax" for RGB
+/// color values, it would be described as `#42A5F5`.
 ///
 /// Here are some ways it could be constructed:
 ///
 /// ```dart
-/// Color c1 = const Color(0xFF42A5F5);
-/// Color c2 = const Color.fromARGB(0xFF, 0x42, 0xA5, 0xF5);
-/// Color c3 = const Color.fromARGB(255, 66, 165, 245);
-/// Color c4 = const Color.fromRGBO(66, 165, 245, 1.0);
+/// Color c1 = const Color.from(alpha: 1.0, red: 0.2588, green: 0.6471, blue: 0.9608);
```

Comment:
style nit:

```
const c1 = Color.from(....)
const c2 = ... 
```

Reply:
Thanks, done!

Reply:
Had to revert this :(, filed https://github.com/flutter/flutter/issues/160884.